models.py

```diff
@@ -232,6 +232,13 @@ def bank_description_check(form, field):
             raise ValidationError('"{}" is not a list/array'.format(FIELDS))
         fields_check(item[FIELDS])
 
+def _formatter(view, context, model, name):
```

Comment:
this formatter is not required, it will get updated on the 'on_model_change' function anyhow

Reply:
deleted the formatter